ReactWindows/ReactNative/Modules/Internationalization/InternationalizationUtil.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Globalization;
+using Windows.Storage;
+
+namespace ReactNative.Modules.Internationalization
+{
+    /// <summary>
+    /// Class used for right to left preferences.
+    /// </summary>
+    static class InternationalizationUtil
+    {
+        private const string AllowRTL = "RCTI18nUtil_allowRTL";
+        private const string ForceRTL = "RCTI18nUtil_forceRTL";
+
+        static InternationalizationUtil()
+        {
+            if (ApplicationData.Current.LocalSettings.Values[AllowRTL] == null)
+                ApplicationData.Current.LocalSettings.Values[AllowRTL] = false;
```

Comment:
Can we just leave these blank and do null coalescing when reading out of LocalSettings below?